.github/scripts/setup_conda.sh

```diff
@@ -33,8 +33,11 @@ conda update --yes conda
 # https://github.com/sherpa/sherpa/pull/794#issuecomment-616570995 )
 # the XSPEC-related channels are only added if needed
 #
+if [ -n "${XSPECVER}" ]; then
+ conda config --add channels conda-forge
```

Comment:
So, presumably there's something in xspec-modelsonly that uses conda-forge ? I'm asking mainly to make it look like I've thought about this ;-)

Reply:
Yes. The default channel does not have a new enough version of gfortran to build Xspec for macOS (as newer versions of xspec requires fgsl which relies on C_LOC support in gfortran ~5.x). So, we need to jump to conda-forge for at least macOS. 
Since we made the jump for both in CIAO (and based on it not seeming to be an issue when asked a few PR meetings ago), I made the jump to conda-forge for both Linux and macOS. 
It actually makes things easier on CM because we can use the same xspec package for CIAO and Standalone Sherpa testing.

Reply:
kewlio.

As the kids don't say. But should.